website/docs/docs/deploy/job-notifications.md

```diff
@@ -111,3 +124,167 @@ If you're logged out or the Slack app/website is closed, you must authenticate b
+
+1. Complete the field defining the Slack workspace you want to integrate with dbt.
+    <Lightbox src="/img/docs/dbt-cloud/define-workspace.png" width="60%" title="Define the workspace"/>
+
+2. Sign in with an existing identity or use the email address and password.
+3. Once you have authenticated successfully, accept the permissions.
+    <Lightbox src="/img/docs/dbt-cloud/accept-permissions.png" width="65%" title="Allow dbt access to Slack"/>
+
+#### Configure Slack notifications
+In this step, you'll configure the Slack channel you want to receive job notifications from.
+
+1. Select your profile icon and then click on **Notification settings**.
+2. Select **Slack notifications** in the left sidebar.
+3. Select the **Notification channel** you want to receive the job run notifications from the dropdown.
+    <Lightbox src="/img/docs/deploy/example-notification-slack-channels.png" width="100%" title="Example of the Notification channel dropdown"/>
+4. Select the **Environment** for the jobs you want to receive notifications about from the dropdown.
+5. Click **Edit** to configure the Slack notification settings. Choose one or more of the run statuses for each job you want to receive notifications about.
+6. When you're done with the settings, click **Save**.
+   - To send alerts to another Slack channel, select another **Notification channel** from the dropdown, **Edit** those job notification settings, and **Save** the changes.
+   - To set up alerts on jobs from a different environment, select another **Environment** from the dropdown, **Edit** those job notification settings, and **Save** the changes.
+    <Lightbox src="/img/docs/deploy/example-slack-notification-settings-page.png" width="100%" title="Example of the Slack notifications page"/>
+
+That's it! Your Slack channel is now set up to receive dbt job notifications at the account level. This integration is now available throughout the account for all licensed users.
+
+#### Disable the Slack integration
+In this step, you'll disable the Slack integration and remove the account-level Slack credentials. You can always re-enable the integration by following the [Set up the Slack integration](#set-up-the-slack-integration-1) steps.
+
+1. Select **Account settings** and on the **Integrations** page, scroll to the **OAuth** section.
+2. Click the x icon (on the far right of the Slack integration) and click **Unlink**.
+    - This removes the account-level Slack credentials. All Slack notifications that rely on the account-level integration will stop sending.
+    - If any legacy, user-linked Slack integrations still exist, those notifications may continue until the legacy link is removed. We recommend migrating to the new account-level app and removing legacy links.
+
+### Microsoft Teams notifications <Lifecycle status="private_beta" />
+
+:::info
+Configuring Microsoft Teams notifications is currently in private beta. To request access, contact dbt Support or your account manager.
```

Comment:
Maybe more of a team question, but do we want to include callouts and pills? I thought the pills were links to reduce this kind of doubling down. I'm totally ok if we keep doing this, but wanted to be sure we're on the same page as a team
```suggestion
Configuring Microsoft Teams notifications is currently in [private beta](/docs/dbt-versions/product-lifecycles#the-dbt-platform). To request access, [contact dbt Support](/docs/dbt-support#dbt-platform-support) or your account manager.
```